PULL_REQUEST_TEMPLATE

```diff
@@ -2,27 +2,20 @@
 
 Please make sure the following is true:
 
-- [ ] This is not a duplicate of another PR
-- [ ] The correct target branch selected?
-- Breaking changes
-  - [ ] No existing features have been broken (without good reason)
-  - [ ] PR introduces breaking changes
-- [ ] Commit messages are meaningful
-- [ ] The behaviour is as the documentation describes, or you updated the docs
-- Tests
-  - [ ] Tests have been added/updated for new/modified unit-testable functions/helpers
-  - [ ] Test were run and did pass
-- [ ] The linter was run and did pass
+- [ ] I gave the PR a meaningful name
+- [ ] I checked that the correct target branch is selected
+- [ ] I rebased the branch on the target branch and it can be merged
+- [ ] I ran the linter and it did pass
+- [ ] I checked for unused code / dead code / debug code
+- [ ] I checked that variables/functions have meaninful names
+- [ ] I checked that the behaviour is as the documentation/task describes and I tested it
+- [ ] I updated the docs / specifications if possible
+- [ ] I could explain all that code when someone wakes me up at 3am
+- [ ] I checked that the code considers failures and not just the happy path
+- [ ] There are no new dependecies
```

Comment:
**[suggestion]** Vll. noch die optionalen Felder, wo man uU etwas angeben muss auch schon als template einfügen?

```
- [ ] There are no new dependecies OR I listed them here:
  - If there are some...
  - ...list them here
- [ ] PR introduces no breaking changes OR I described them here:
  - If it introduces some...
  - ...list them here
```

Reply:
Ich würde die Auflistung/Erklärungen nach unten bringen unter `## Other information/comments (e.g. reasons why points are not checked from above)`

```
- [ ] There are no new dependencies OR I listed them and explained them below
- [ ] PR introduces no breaking changes OR I listed them and described them below
  ```
  
  Dann bleibt die Liste zum abhaken immer gleich und nur unten sind dann weitere Informationen.
  Was denkt Ihr?

Reply:
Das ist zumindest so, wie wir uns im Workshop geeinigt haben, damit die Liste eben einheitlich bleibt. Letztendlich bin ich für beides offen.